### ISSUE TITLE
Climate entity: verify hvac_mode feature mapping

### ISSUE THREAD

**Issue by zehnm** [unfoldedcircle/integration-home-assistant#11]
Test that the `hvac_mode` features are correctly mapped:
- [x] `off`
- [x] `heat`
- [ ] `cool`
It's implemented based on the HA docs, but not fully tested with real devices.

https://developers.home-assistant.io/docs/core/entity/climate#hvac-modes

**Comment by chrissi323** [comment]
I tested with Homematic IP, `Off `and `Heat `works, but I can't set it to `Auto` which ist what you want to use most of the timw with these so they work independently with the preprogramed heating times, even when HA or the Homematic hub is down. So for now I can disable heating or set it to a fixed temperatue.

I only tested with the emulator of the linux-vm as I don't have the real remote yet.

**Comment by chrissi323** [comment]
A year later i now have my Remote 3. Same as a I worte in my previous post. Could you please implement HVACMode.AUTO?

**Comment by sukumarkonduru** [comment]
Yes. We need auto to be supported. It is strange behavior if I chose auto
